.github/workflows/ci.yml

```diff
@@ -0,0 +1,100 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    services:
+      db:
+        image: postgres:9.4
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_HOST_AUTH_METHOD: trust
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    
+    env:
+      RAILS_ENV: test
+      CC_TEST_REPORTER_ID: ${{ secrets.CC_TEST_REPORTER_ID }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.7.2
+      - name: Configure Ruby env
+        run: |
+          echo "gem: --no-document" > ~/.gemrc
+          bundle config set clean 'true'
+      - name: Ruby gem cache
+        uses: actions/cache@v2
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
```

Comment:
Something I noticed is that it generated different hashes for files in the push/PR runs ("Ruby gem cache" & "JS package cache" steps). 

I think we need to check this behavior to see if this happens for every PR (doesn't find the cache key in the first run), or if this happened here because it's the first PR that uses GA CI.

[PR - 1st run "Ruby gem cache" step](
https://github.com/rootstrap/rails_api_base/runs/6011073300?check_suite_focus=true), it didn't find the gems cache for the given key
<img width="980" alt="imagen" src="https://user-images.githubusercontent.com/80283482/163234034-b018e911-5b96-46e8-a89d-f4f92a938b24.png">

[PR - 2nd run "Ruby gem cache" step](https://github.com/rootstrap/rails_api_base/runs/6011213086?check_suite_focus=true), it found the cache key because it was generated in the first run
<img width="990" alt="imagen" src="https://user-images.githubusercontent.com/80283482/163234332-37095c6b-fc4e-4ec6-81c6-2414b8a07279.png">

[Push run - "Ruby gem cache" step](https://github.com/rootstrap/rails_api_base/runs/6010763308?check_suite_focus=true), the cache key is different from the one used in the PR
<img width="1023" alt="imagen" src="https://user-images.githubusercontent.com/80283482/163234663-b6cfe128-9727-4826-aa7c-02b595c0dac6.png">

Reply:
Can you try this?
```yaml
- name: Setup Ruby
   uses: ruby/setup-ruby@v1
   with:
     bundler-cache: true
```
https://github.com/ruby/setup-ruby#caching-bundle-install-automatically